### PR TITLE
Enable monitoring

### PR DESCRIPTION
**Pull request by vipin-dfe**
### Context
RSM Migration to AKS production environment and it requires monitoring. 

### Changes proposed in this pull request
Enable azure monitoring, prometheus metric scrapping and status cake alerting

### Guidance to review
Check if azure monitoring is enabled for cloud resources. 
Check if the metrics can be scrapped from the prometheus metrics endpoint.

### Link to Trello card
[2161-rsm-enable-monitoring](https://trello.com/c/wBMRU68q)

### Checklist

- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: terraform/application/application.tf [hunk 1]
```diff
@@ -53,18 +54,19 @@ module "main_worker" {
...
+  docker_image                 = var.docker_image
+  replicas                     = var.worker_replicas
+  max_memory                   = var.worker_memory_max
+  cluster_configuration_map    = module.cluster_data.configuration_map
+  kubernetes_config_map_name   = module.application_configuration.kubernetes_config_map_name
+  kubernetes_secret_name       = module.application_configuration.kubernetes_secret_name
+  command                      = ["/bin/sh", "-c", "bundle exec sidekiq -C config/sidekiq.yml"]
+  probe_command                = ["pgrep", "-f", "sidekiq"]
+  enable_logit                 = var.enable_logit
+  enable_prometheus_monitoring = var.enable_monitoring
```
Comment (RMcVelia) [inline terraform/application/application.tf:71]: I couldn't see where rsm has a metrics endpoint?

File: terraform/application/application.tf [hunk 2]
```diff
@@ -37,9 +37,10 @@ module "web_application" {
   service_name = var.service_name
   probe_path   = "/health"
 
-  cluster_configuration_map  = module.cluster_data.configuration_map
-  kubernetes_config_map_name = module.application_configuration.kubernetes_config_map_name
-  kubernetes_secret_name     = module.application_configuration.kubernetes_secret_name
+  cluster_configuration_map    = module.cluster_data.configuration_map
+  kubernetes_config_map_name   = module.application_configuration.kubernetes_config_map_name
+  kubernetes_secret_name       = module.application_configuration.kubernetes_secret_name
+  enable_prometheus_monitoring = var.enable_monitoring
```
Comment (RMcVelia) [inline terraform/application/application.tf:43]: I couldn't see where rsm has a metrics endpoint?

File: Makefile [hunk 3]
```diff
@@ -242,6 +242,11 @@ deploy-arm-resources: arm-deployment ## Validate ARM resource deployment. Usage:
 
 validate-arm-resources: set-what-if arm-deployment ## Validate ARM resource deployment. Usage: make domains validate-arm-resources
 
+action-group-resources: composed-variables set-azure-account # make env action-group-resources ACTION_GROUP_EMAIL=notificationemail@domain.com . Must be run before setting enable_monitoring=true for each subscription
```
Comment (saliceti) [inline Makefile:245]: composed-variables may not be needed

File: terraform/application/application.tf (hunk 2 above)
Comment (vipin-dfe) [reply terraform/application/application.tf]: Disabled prometheus monitoring. Code updated.

File: Makefile (hunk 3 above)
Comment (vipin-dfe) [reply Makefile]: You are correct. We donot need it anymore. Code updated.

**Review by saliceti — APPROVED** [review]

**Comment by neillturner** [comment]
[RSM: migrate production environment](https://trello.com/c/EfDlDX1V/2160-rsm-migrate-production-environment)

**Pull request merged**